components/openindiana/nwam-manager/Makefile

```diff
@@ -85,3 +87,14 @@ build: $(BUILD_32)
 install: $(INSTALL_32)
 
 download:: $(SOURCE_DIR)/.downloaded
+
+REQUIRED_PACKAGES += gnome/config/gconf
+REQUIRED_PACKAGES += library/desktop/atk
+REQUIRED_PACKAGES += library/desktop/gtk2
+REQUIRED_PACKAGES += library/desktop/libglade
+REQUIRED_PACKAGES += library/glib2
+REQUIRED_PACKAGES += library/gnome/gnome-libs
```

Comment:
Okay, I've rebuilt the REQUIRED_PACKAGES list for openindiana/nwam-manager on x86 hipster.  It looks just like yours now.  The component still builds on both hardware platforms.